spec/rubocop/cop/rails/file_path_spec.rb

```diff
@@ -85,6 +105,23 @@
           "#{Rails.root}/app/models/goober"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `Rails.root.join('path/to')`.
         RUBY
+
+        expect_correction(<<~'RUBY')
+          "#{Rails.root.join("app/models/goober")}"
+        RUBY
+      end
+    end
+
+    context 'when using Rails.root called by double quoted string that ends with string interpolation' do
+      it 'registers an offense' do
+        expect_offense(<<~'RUBY')
+          "#{Rails.root}/a/#{b}"
+          ^^^^^^^^^^^^^^^^^^^^^^ Prefer `Rails.root.join('path/to')`.
+        RUBY
+
+        expect_correction(<<~'RUBY')
+          "#{Rails.root.join("a/#{b}")}"
+        RUBY
       end
     end
```

Comment:
> I've just merge https://github.com/rubocop/rubocop-rails/pull/989. Can you rebase with the latest master branch and add a expect_correction test for that?

Thanks for the quick merge! Actually, I already have a test case for that here 👌 
Anyway, I will rebase and try to resolve the conflict.